secator/cli_helper.py

```
@@ -302,8 +322,27 @@
+			if enable_memray:
+				process = psutil.Process()
+				console.print(
+					f"[bold yellow3]Initial RAM Usage: {process.memory_info().rss / 1024 ** 2} MB[/]"
+				)
+			item_count = 0
+			runner = runner_cls(
+				config, inputs, run_opts=opts, hooks=hooks, context=context
+			)
+			for item in runner:
+				del item
+				item_count += 1
```

Ruff (0.11.9)

Use `enumerate()` for index variable `item_count` in `for` loop

(SIM113)